### ISSUE TITLE
.swiftformat file should end with a linefeed in all configurations

### ISSUE THREAD

**Issue by rudifa** [nicklockwood/SwiftFormat#293]
Currently, a `.swiftformat` file that ends with a `--disable` or an `--enable` line has no final linefeed. 

I propose the following patch to remedy this admittedly minor defect.

```
---
 Sources/Arguments.swift | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/Sources/Arguments.swift b/Sources/Arguments.swift
index e96ff02..829d491 100644
--- a/Sources/Arguments.swift
+++ b/Sources/Arguments.swift
@@ -230,10 +230,15 @@ func serialize(options: Options,
     if let rules = options.rules {
         optionSets.append(Options(rules: rules))
     }
-    return optionSets.map {
+    var optionStrings: [String] = optionSets.map {
         let arguments = argumentsFor($0, excludingDefaults: excludingDefaults)
         return serialize(arguments: arguments, separator: separator)
-    }.joined(separator: separator)
+    }
+    optionStrings = optionStrings.filter { !$0.isEmpty }
+    if optionStrings.count > 0 {
+        optionStrings.append("")
+    }
+    return optionStrings.joined(separator: separator)
 }
 
 // Serialize arguments
-- 
```

[rf_patch__newline_at_swiftformat_eof.patch.zip](https://github.com/nicklockwood/SwiftFormat/files/2311938/rf_patch__newline_at_swiftformat_eof.patch.zip)

**Comment by nicklockwood** [comment]
Ah, good catch - thank you!

If it’s not too much trouble, could you submit your patch as a pull request? Then I’ll merge it into the next release.

**Comment by nicklockwood** [comment]
@rudifa your fix has been included in version 0.35.4. Thanks!
